### PR TITLE
Local setup

### PR DESCRIPTION
**Pull request by sinclairs**
Makefile script is a pattern used in [eslint](https://github.com/eslint/eslint/blob/main/Makefile.js)

@msenkiw it would be good if you could have a look at this PR, specifically around the thin / thick mode. My understanding is that thick mode would only be required for older DB versions and could probably be removed for our purposes?

### CODE REVIEW

File: Makefile.mjs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (armoj) [inline Makefile.mjs:1]: is this needed if `execa` is declared in the `package.json` file?

File: Makefile.mjs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import {$} from 'execa';
+
+const commands = new Set(process.argv);
+
+if (commands.has('init')) {
```
Comment (armoj) [inline Makefile.mjs:6]: Does this file structure mean that `node Makefile.mjs create-secrets create-queue start create-bucket init 
 upload-file  create-parameters ` is a legal command? Is this the intent?

I can see that all commands if blocks have an `await` so things won't trip up, but should they just return/break after each one if chaining is not intended here?

**Review by armoj — CHANGES_REQUESTED** [review]

File: Makefile.mjs (hunk 1 above)
Comment (sinclairs) [reply Makefile.mjs:1]: As per discussion: It complains that execa is only included in `devDependencies`, which is fine, as it **_is_** a dev dependency

File: Makefile.mjs (hunk 2 above)
Comment (sinclairs) [reply Makefile.mjs:8]: As per discussion: That command style is legal, however it hasn't been documented and isn't supported.

*I did find it useful during development

**Review by armoj — APPROVED** [review]

**Review by msenkiw — APPROVED** [review]

**Pull request merged**